### PR TITLE
Enable compilation on app extensions

### PR DESCRIPTION
**Pull request by magnusottosson**
Maybe not the best solution but this enables compilation on app extensions.

### CODE REVIEW

**Comment by tsheaff** [comment]
How does this fix the problem?  You're still referring to `[UIApplication sharedApplication]`

File: SexyTooltip/SexyTooltip.m [hunk 1]
```diff
@@ -75,7 +75,9 @@ - (id)initWithAttributedString:(NSAttributedString *)attrStr
 {
     self = [self init];
     if (self) {
-        containerView = containerView ?: [[UIApplication sharedApplication].windows firstObject];
+
+    	NSAssert(containerView, @"Container view can not be nil.");
```
Comment (tsheaff) [inline SexyTooltip/SexyTooltip.m]: spaces not tabs

**Comment by tsheaff** [comment]
Please bump the podspec version to `1.2.1` and fix the indentation issues.  Then it looks OK.

Thanks!

**Comment by magnusottosson** [comment]
Done!

**Comment by tsheaff** [comment]
@permagnus nice. Merged, pushed tag `1.2.1`, and did `pod trunk push` so it's live if you update your Podfile.

**Pull request merged**